src/defaultImages.ts

```diff
@@ -1,11 +1,11 @@
 export const defaultImages = {
-    balloon: require('../images/balloon.svg'),
-    bug: require('../images/bug.svg'),
-    compass: require('../images/compass.svg'),
-    ghost: require('../images/ghost.svg'),
-    iconWhiteTransparent: require('../images/icon-white-transparent.png'),
-    link: require('../images/link.svg'),
-    snorkeling: require('../images/snorkeling.svg'),
-    defaultUser: require('../images/assets/defaultuser.png'),
-    felfeleAssistant: require('../images/felfele-assistant.png'),
+    balloon: require('../images/balloon.svg') as number,
```

Comment:
Good idea. I considered using `ImageRequireSource` from the type definition of React Native. However it is just an alias, so there were no real advantage to do that and it transitively also made lots of things a dependency of React that is not required otherwise.